Wason_2-4-6_box/src/Wason.ino

```diff
@@ -0,0 +1,743 @@
+#include "SevenSegmentTM1637.h"
```

Comment:
There is `wason.ino` (lower case) in this directory already. How is it related to the new file ?

Reply:
it's the incomplete version that i didn't remove (the difference between lowercase and uppercase is because of some IDE stuff in my directories, nothing significant).